contents/handbook/small-teams/marketing/objectives.mdx

```diff
@@ -1,18 +1,36 @@
 * **Objective**:
-    * Expand PostHog’s top of funnel awareness across all our core _individual_ products - product analytics, session recording, feature flags, and A/B testing.
+    * Increase word-of-mouth growth by:
+      1. being an authority and vital resource on building successful products.
+      2. creating a world-class community experience.
 
-* **Key Results:**
-    * Increase [overall weekly website traffic](https://app.posthog.com/insights/hg6kfoIG) to 27k unique visitors per week (filtering out viral spikes).
-    * Increase [‘first user in org’ signups](https://app.posthog.com/insights/EwWz7QuV) to 600 per week without decreasing the percentage that are ICPs (20-25%).
-    * Make Tutorials our [3rd most visited section](https://app.posthog.com/insights/n8b9SHgd) (after Docs and Blogs)
-    * Increase number of [subscribers](https://newsletter.posthog.com/publish/subscribers) to Product for Engineers to 1,500
-    * Get [200 orgs](https://docs.google.com/spreadsheets/d/1ryDGzXcG0OSH-GoX-zlI2CKwNu-6Zivi7EM32WLdids/edit#gid=0) to sign up to the startup deal
-    * Increase [28 day YouTube watch time](https://studio.youtube.com/channel/UCn4mJ4kK5KVSvozJre645LA?c=UCn4mJ4kK5KVSvozJre645LA) to 227 hours
+* **Goals:**
+    * Nail Twitter and content distribution (**Owner:** TBD, maybe Charles?)
+    * Nail internal content discovery and engagement (**Owner:** Ian with Website & Docs)
+    * Make PostHog for Startups the default choice for technical founders (**Owner:** Joe)
+    * Scale the newsletter to thousands of engaged subscribers (**Owner:** Andy)
+    * Create and execute a community plan (**Owner:** Lior)
```

Comment:
I think "create a plan" as a goal needs to become "<the plan>" over the next few days. *zero* problem if you change it once you start / make a mistake. Worried otherwise we'll be in paralysis by analysis mode.

Reply:
Sounds good to me. No idea what this entails, but excited to figure it out!